src/System.Reflection.DispatchProxy/src/Resources/Strings.resx

```diff
@@ -70,4 +70,7 @@
  <data name="BaseType_Cannot_Be_Abstract" xml:space="preserve">
     <value>The base type '{0}' cannot be abstract.</value>
   </data>
+  <data name="PlatformNotSupported_ReflectionDispatchProxy" xml:space="preserve">
+    <value>This operation is only supported where Reflection.Emit is supported.</value>
```

Comment:
Please state where it is suported (or where it is not supported). As it is the message is nearly useless :)